test/bdd/features/healthcheck_api.feature

```diff
@@ -11,8 +10,9 @@ Feature: health check
 
   Scenario Outline:
     When an HTTP GET is sent to "<url>"
-    Then the JSON path "status" of the response equals "success"
+    Then the JSON path "<respKey>" of the response equals "<respKeyVal>"
     Examples:
-      | url                                     |
-      | http://localhost:8070/healthcheck       |
-      | http://localhost:8069/healthcheck       |
+      | url                                             | respKey       | respKeyVal                                      |
+      | http://localhost:8070/healthcheck               | status        | success                                         |
+      | http://localhost:8060/healthcheck               | status        | success                                         |
+      | http://localhost:8060/didcomm/invitation        | @type         | https://didcomm.org/didexchange/1.0/invitation  |
```

Comment:
Will this become out-of-band?

Reply:
Yeah, internally gonna switch from `didexchange` to `oob`. For now, added generic `didcomm` term in the path.